### DISCUSSION TITLE
Optionally disable multi selection feature

### DISCUSSION THREAD

**Discussion by tbarusseau** [aevyrie/bevy_mod_picking#172] [Ideas]
Hello! Great plugin, thanks a lot for what you've build and for keeping it up to date with bevy.

Is there a way to disable the multiple selection?
Having it on is not actually a worry because you can still handle `PickingEvent`s without support for multi selection, but since there is currently a built-in graphical feedback it would fell weird to the user if he can't actually do anything with multiple selected entities.

Cheers!

**Comment by aevyrie** [comment]
This isn't possible on main/crates at the moment. It should be available in the refactor branch. Your best bet is to make your own version of `selection.rs`, it's a super simple module to replace with your own implementation. https://github.com/aevyrie/bevy_mod_picking/blob/main/src/selection.rs

**Reply by aevyrie** [reply]
It is now possible to disable the default multiselect buttons, which would result in the desired behavior. https://github.com/aevyrie/bevy_mod_picking/blob/b7d71343ebe09cbb9653eb31db4ddf6ad6950f3a/crates/bevy_picking_selection/src/lib.rs#L24
